[Lite version] "No database configured" error when trying to play song.

<!-- Use Discord for questions: https://discord.gg/sbySMS7m3v -->

## Please describe the problem you are having in as much detail as possible:
When I run the lite version with "make lite" I get a "Error: No database configured" message despite me not running it with the 'db' option.
## Include a reproducible code sample here, if possible, the snippet of code which is throwing the error or which you presume is giving issues:

```js
[13:11:2023 - 23:12] [ERROR] | [FATAL] Possibly Uncaught Exception
        Reason: Error: No database configured

uncaughtException

```
## If you have your own setup, without the original docker configuration, please provide:

I have both DATABASE and DATABASE_URL set to "" in the .env file
Running the latest master branch with Node.js v21.4.0 on Arch Linux.

--- a/dashboard/src/components/ModalShortcut.tsx
+++ b/dashboard/src/components/ModalShortcut.tsx
@@ -4,12 +4,15 @@ import { Container } from '@nextui-org/react';
 import ModalCloseIcon from '@/assets/icons/modal-close.svg';
 import Key from '@/components/shortcuts/Key';
 import ShortcutEntry from './shortcuts/ShortcutEntry';
+import { getAllShortcuts } from '@/libs/kbdsrct';
 
 export default function ModalShortcut({
     open,
     onClose,
     modalContainerProps = {},
 }: IModalShortcutProps) {
+    const allShortcuts = getAllShortcuts();
+
     return (
         <Modal open={open} {...modalContainerProps}>
             <Container
@@ -69,13 +72,18 @@ export default function ModalShortcut({
                     }}
                 >
                     <div>
-                        <h2>Playback</h2>
-                        <ShortcutEntry name="Play / Pause" comb={['Space']} />
-                        <ShortcutEntry name="Next Track" comb={['Alt', '→']} />
-                        <ShortcutEntry
-                            name="Previous Track"
-                            comb={['Alt', '←']}
-                        />
+                        {allShortcuts.map((s) => (
+                            <>
+                                <h2>{s.category}</h2>
+                                {s.shorcuts.map((h) => (
+                                    <ShortcutEntry
+                                        key={`${h.category}-${h.description}`}
+                                        description={h.description}
+                                        comb={h.combDisplay}
+                                    />
+                                ))}
+                            </>
+                        ))}
                     </div>
                 </div>
             </Container>

--- a/dashboard/src/components/ModalShortcut.tsx
+++ b/dashboard/src/components/ModalShortcut.tsx
@@ -79,7 +79,7 @@ export default function ModalShortcut({
                                     <ShortcutEntry
                                         key={`${h.category}-${h.description}`}
                                         description={h.description}
-                                        comb={h.combDisplay}
+                                        comb={h.combDisplay ?? h.comb}
                                     />
                                 ))}
                             </>

--- a/dashboard/src/components/shortcuts/ShortcutEntry.tsx
+++ b/dashboard/src/components/shortcuts/ShortcutEntry.tsx
@@ -2,7 +2,7 @@ import { IShortcutEntryProps } from '@/interfaces/components/Shortcuts';
 import Key from '@/components/shortcuts/Key';
 
 export default function ShortcutEntry({
-    name,
+    description,
     comb = [],
 }: IShortcutEntryProps) {
     return (
@@ -13,15 +13,13 @@ export default function ShortcutEntry({
                 alignItems: 'center',
             }}
         >
-            <div>{name}</div>
+            <div>{description}</div>
             <div
                 style={{
                     display: 'flex',
                 }}
             >
-                {comb.map((v, i) => (
-                    <Key key={i}>{v}</Key>
-                ))}
+                {comb?.map((v, i) => <Key key={i}>{v}</Key>)}
             </div>
         </div>
     );

--- a/dashboard/src/interfaces/components/Shortcuts.ts
+++ b/dashboard/src/interfaces/components/Shortcuts.ts
@@ -3,6 +3,6 @@ export interface IKeyProps {
 }
 
 export interface IShortcutEntryProps {
-    name?: React.ReactNode;
+    description?: React.ReactNode;
     comb?: IKeyProps['children'][];
 }

--- a/dashboard/src/interfaces/kbdsrct.ts
+++ b/dashboard/src/interfaces/kbdsrct.ts
@@ -1,4 +1,13 @@
 export interface IKbdsrct {
     comb: string[];
     cb: () => void;
+    category?: string;
+    description?: string;
+    hide?: boolean;
+    combDisplay?: string[];
+}
+
+export interface IShorcutGroup {
+    category: string;
+    shorcuts: IKbdsrct[];
 }

--- a/dashboard/src/layouts/AppLayout.tsx
+++ b/dashboard/src/layouts/AppLayout.tsx
@@ -79,6 +79,7 @@ const AppLayout: PageLayout = ({
     const modalShortcutKbdsrct = {
         comb: ['Control', '/'],
         cb: () => setModalShortcutOpen((v) => !v),
+        hide: true,
     };
 
     useEffect(() => {

--- a/dashboard/src/libs/kbdsrct.ts
+++ b/dashboard/src/libs/kbdsrct.ts
@@ -1,4 +1,4 @@
-import { IKbdsrct } from '@/interfaces/kbdsrct';
+import { IKbdsrct, IShorcutGroup } from '@/interfaces/kbdsrct';
 
 const kbdsrcts: IKbdsrct[] = [];
 
@@ -86,4 +86,27 @@ export function execKbdsrct(keypresses: string[]) {
     }
 
     return hasSrct;
+}
+
+export function getAllShortcuts() {
+    const allShortcuts: IShorcutGroup[] = [];
+
+    for (const k of kbdsrcts) {
+        if (k.hide) continue;
+
+        const ref = allShortcuts.find((v) => v.category === k.category);
+
+        if (!ref) {
+            allShortcuts.push({
+                category: k.category,
+                shorcuts: [k],
+            });
+
+            continue;
+        }
+
+        ref.shorcuts.push(k);
+    }
+
+    return allShortcuts;
 }

--- a/dashboard/src/pages/servers/[id]/player.tsx
+++ b/dashboard/src/pages/servers/[id]/player.tsx
@@ -445,28 +445,37 @@ const Player: NextPageWithLayout = () => {
         // literally a space
         comb: [' '],
         cb: togglePlayPause,
+        category: 'Playback',
+        description: 'Play / Pause',
+        combDisplay: ['Space'],
     };
 
     const arrowLeftPrev = {
         comb: ['Alt', 'ArrowLeft'],
         cb: handlePrevious,
+        category: 'Playback',
+        description: 'Previous Track',
+        combDisplay: ['Alt', '←'],
     };
 
     const arrowRightNext = {
         comb: ['Alt', 'ArrowRight'],
         cb: handleNext,
+        category: 'Playback',
+        description: 'Next Track',
+        combDisplay: ['Alt', '→'],
     };
 
     const registerAllSrct = () => {
         registerKbdsrct(spacePP);
-        registerKbdsrct(arrowLeftPrev);
         registerKbdsrct(arrowRightNext);
+        registerKbdsrct(arrowLeftPrev);
     };
 
     const unregisterAllSrct = () => {
         unregisterKbdsrct(spacePP);
-        unregisterKbdsrct(arrowLeftPrev);
         unregisterKbdsrct(arrowRightNext);
+        unregisterKbdsrct(arrowLeftPrev);
     };
 
     useEffect(() => {

--- a/djs-bot/interactions/autoqueue.js
+++ b/djs-bot/interactions/autoqueue.js
@@ -1,6 +1,5 @@
 const SlashCommand = require("../lib/SlashCommand");
 const { ccInteractionHook } = require("../util/interactions");
-const { autoQueueEmbed } = require("../util/embeds");
 const { updateControlMessage } = require("../util/controlChannel");
 
 const command = new SlashCommand()

--- a/djs-bot/package.json
+++ b/djs-bot/package.json
@@ -1,7 +1,7 @@
 {
 	"description": "",
 	"main": "index.js",
-	"version": "5.3.2",
+	"version": "5.4.0",
 	"name": "discord-musicbot",
 	"scripts": {
 		"guild": "node scripts/guild",

--- a/djs-bot/util/controlChannel.js
+++ b/djs-bot/util/controlChannel.js
@@ -23,7 +23,8 @@ const getControlChannelMessage = async (guildId) => {
 
 	const client = getClient();
 
-	if (!client.db) throw new Error("No database configured");
+	// no db? simply ignore
+	if (!client.db) return;
 
 	const { controlChannelId, controlChannelMessageId } =
 		(await client.db.guild.findFirst({
@@ -66,7 +67,8 @@ const setDbControlChannel = async ({ guildId, channelId, messageId } = {}) => {
 		);
 	else deleteControlChannelMessage(guildId);
 
-	if (!client.db) throw new Error("No db configured");
+	// no db? simply ignore
+	if (!client.db) return;
 
 	await client.db.guild.upsert({
 		where: {
@@ -103,7 +105,8 @@ const handleMessageDelete = async (message) => {
 
 	const client = getClient();
 
-	if (!client.db) throw new Error("No database configured");
+	// no db? simply ignore
+	if (!client.db) return;
 
 	await client.db.guild.update({
 		where: {

